sentry-uitest/sentry-uitest-android-benchmark/build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.extensions.DetektExtension
+import net.ltgt.gradle.errorprone.errorprone
+
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    id(Config.QualityPlugins.errorProne)
+    id(Config.QualityPlugins.gradleVersions)
+    id(Config.QualityPlugins.detektPlugin)
+}
+
+configure<JavaPluginExtension> {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+android {
+    compileSdk = Config.Android.compileSdkVersion
+
+    defaultConfig {
+        applicationId = "io.sentry.uitest.android.benchmark"
+        minSdk = Config.Android.minSdkVersionNdk
+        targetSdk = Config.Android.targetSdkVersion
+        versionCode = 1
+        versionName = "1.0.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Let's extract this to a const in the Config class, there are more usages across packages